capstone/src/main/java/com/google/sps/servlets/YoutubeUsernameServlet.java

```diff
@@ -30,20 +30,25 @@
 import org.json.simple.JSONObject;    
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.io.UnsupportedEncodingException; 
+import java.net.MalformedURLException; 
+import java.net.URL; 
+import java.net.URLEncoder; 
 
 /** Servlet that converts a youtube username to a channelID. */
-@WebServlet("/username_servlet")
-public class UsernameConverter extends HttpServlet {
-  private static final String URL = " https://www.googleapis.com/youtube/v3/channels?key=";
+@WebServlet("youtube_username_servlet")
```

Comment:
nit:
```suggestion
@WebServlet("/youtube_username_servlet")
```

Reply:
addressed in "address Chris's and Daniels nits"